packages/integration-karma/test/template/attributes/index.spec.js

```diff
@@ -30,3 +30,17 @@ it(`should remove the existing attribute if set to null`, () => {
         expect(elm.shadowRoot.querySelector('div').getAttribute('title')).toBe(null);
     });
 });
+
+it(`should remove the existing attribute if set to undefined`, () => {
+    const elm = createElement('x-test', { is: Test });
+    elm.attr = undefined;
+    document.body.appendChild(elm);
+
+    expect(elm.shadowRoot.querySelector('div').getAttribute('title')).toBe(null);
+
+    elm.attr = undefined;
```

Comment:
This shouldn't trigger a re-render as far as I know.

Reply:
fixed.